optimum/exporters/tasks.py

```diff
@@ -1770,6 +1770,7 @@ def _infer_task_from_model_name_or_path(
         revision: Optional[str] = None,
         cache_dir: str = HUGGINGFACE_HUB_CACHE,
         token: Optional[Union[bool, str]] = None,
+        library_name: Optional[str] = None,
```

Comment:
I see you're adding `library_name` to the signature but shouldn't this argument be specified in the method's calls as well ?

Reply:
I just want to get confirmation that it is reliable approach before doing that.
In case of optimum itself it also need to provide it to infer_task_from_model too https://github.com/huggingface/optimum/blob/main/optimum/exporters/onnx/__main__.py#L259

Reply:
yeah you can go ahead and add it where needed, it seems reliable.

Reply:
ok, added.

Only not sure about tflite exported, as I can see for getting export config, it uses library_name="transformers" explicitly, so it means that other library are not supported, right? I also added library_name="transformers" for avoid model type mismatch (e.g. in case of sentence transformers described above, if it will not be provided, model will be loaded as sentence transformer and raises error in some steps later